### ISSUE TITLE
ep issue

### ISSUE THREAD

**Issue by ghost** [AdminConfirmed/EnderPearlCooldown#1]
takes away enderpearl when u use it, it doesnt give the ep back. Can u add that?

**Comment by jaylac2000** [comment]
kinda how e pearls work... lol... this is just to make it so people caNT use them for so much time

**Comment by ItsMax123** [comment]
This is true, can this issue be fixed? all that needs to be added is a line to additem 1 enderpearl to player when event is canceled.

**Comment by jaylac2000** [comment]
lol than fork it

**Comment by ItsMax123** [comment]
um ok i can try
